packages/sit-onyx/src/components/OnyxRipple/OnyxRipple.stories.ts

```diff
@@ -0,0 +1,37 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxRipple from "./OnyxRipple.vue";
+
+/**
+ * OnyxRipple provides visual feedback when a user interacts with the component. Supporting both touch and mouse events.
+ * A standalone OnyxRipple does not show any ripples, since the events have to be handled on the parent component. See OnyxButton how to handle that.
+ */
+const meta: Meta<typeof OnyxRipple> = {
+  title: "Support/Ripple",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxRipple,
+    events: [],
+  }),
+};
+
+export default meta;
+type Story = StoryObj<typeof OnyxRipple>;
+
+/**
+ * Default properties like color and durations can be overridden. Click anywhere to create a ripple.
+ */
+export const Default = {
```

Comment:
The example does not work for me

Reply:
I see, thats because the events are now triggered on the parent. 

shall we link to the OnyxButton story ?
I guess we won't be able to trigger anything exposed by the Ripple component ?

Reply:
Yes thats fine for me